### DISCUSSION TITLE
Does the break package still support BRKEXP?

### DISCUSSION THREAD

**Discussion by pamoroso** [Interlisp/medley#1423] [Q&A]
Does the break package still support `BRKEXP`? Did the integration with Common Lisp remove `BRKEXP` or change its behavior?

Evaluating `BRKEXP` in a break exec seems to always return `NIL` regardless of what form caused the break.

**Comment by masinter** [comment]
Treat this as an Issue in the "environment" project. BREAK and BREAKDOWN should work as expected... we should be able to set breakpoints and stepping for both Common Lisp and Interlisp, and do it as well as Slime and other Common Lisp debugging tools.

**Reply by pamoroso** [reply]
I've created issue #1424.
